src/checkData.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable max-len */
+function validateData(text, caseType) {
+  const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const errors = [];
+
+  if (!text) {
+    errors.push({
+      message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!caseType) {
+    errors.push({
+      message: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (caseType && !availableCases.includes(caseType)) {
+    errors.push({
+      message: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+    });
+  }
+
+  return errors;
```

Comment:
why do you create separate files for all functions?
Just create one file like a `heplers` for the server